lib/command/init.rb

```diff
@@ -128,26 +133,31 @@ def ask_aozoraepub3_path
       end
       print ")\n>"
       while input = $stdin.gets
-        if Helper.os_windows?
-          input.force_encoding(Encoding::Windows_31J).encode!(Encoding::UTF_8)
-        end
-        input = input.strip.gsub(/"/, "")
-        path = File.expand_path(input)
-        case
-        when input == ":keep"
-          aozora_dir = @global_setting["aozoraepub3dir"]
-          if aozora_dir && Narou.aozoraepub3_directory?(aozora_dir)
-            return aozora_dir
-          end
-        when Narou.aozoraepub3_directory?(path)
-          return path
-        when input == ""
-          break
-        end
+        break if input.strip! == ""
+        checked_input = check_aozoraepub3_path(input)
+        return checked_input if checked_input
         print "\n<bold><green>入力されたフォルダにAozoraEpub3がありません。" \
               "もう一度入力して下さい:</green></bold>\n&gt;".termcolor
       end
       nil
     end
+
+    def check_aozoraepub3_path (input)
+      if Helper.os_windows?
+        input.force_encoding(Encoding::Windows_31J).encode!(Encoding::UTF_8)
+      end
+      input = input.gsub(/"/, "")
```

Comment:
__[rubocop]__
* Performance/StringReplacement: Use `delete` instead of `gsub`. [:ref](https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)

<!-- Comment by [SideCI](https://sideci.com) -->